### DISCUSSION TITLE
容器管理 - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1143] [Announcements]
# 容器管理 - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/videos/kpanda/

**Comment by gssjl2008** [comment]
晤，页面自动播放视频功能建议关掉，瞬间5,6个视频同时播放， 体验不好。

**Comment by lemontrees123** [comment]
已关闭，目前已不会自动播放
